README.md

```diff
@@ -149,6 +149,30 @@ This is the command to install the nightly with ROCm 6.2 which might have some p
 
 ```pip install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/rocm6.2```
 
+### Intel GPUs (Windows and Linux)
+
+(Option 1) Intel Arc GPU users can install native PyTorch with torch.xpu support using pip (currently available in PyTorch nightly builds). More information can be found [here](https://pytorch.org/docs/main/notes/get_start_xpu.html)
+  
+1. To install PyTorch nightly, use the following command:
+
+```pip install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/xpu```
+
+2. Launch ComfyUI by running `python main.py`
+
+
+(Option 2) Alternatively, Intel GPUs supported by Intel Extension for PyTorch (IPEX) can leverage IPEX for improved performance.
+
+1. For Intel® Arc™ A-Series Graphics utilizing IPEX, create a conda environment and use the commands below:
+
+```
+conda install libuv
```

Comment:
Is this step necessary?

Reply:
It is necessary if users want to use Intel Extension for Pytorch for better performance on Intel GPUs. 

Currently there is a perf gap in Pytorch 2.5 / Pytorch 2.6 nightly vs. Pytorch 2.3 (also perf regression on Nvidia/AMD too, as reported in https://github.com/pytorch/pytorch/issues/138386)

so it's good to have both options available for Intel GPU users, and show an example of the env setup.

Eventually we would want to make an update to ComfyUI readme when those issues are fixed 😄 

Thanks